compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_memory.rs

```diff
@@ -456,6 +437,28 @@ impl<F: AcirField + DebugToString, Registers: RegisterAllocator> BrilligContext<
         self.codegen_initialize_vector_metadata(vector, size, capacity);
     }
 
+    /// Writes vector metadata (reference count, size, and capacity) into the allocated memory
+    pub(super) fn codegen_initialize_vector_metadata(
```

Comment:
I just moved this after `codegen_initialize_vector` and changed the vis to `super`